keel-core/src/main/kotlin/com/netflix/spinnaker/keel/notifications/NotificationType.kt

```diff
@@ -6,5 +6,9 @@ package com.netflix.spinnaker.keel.notifications
 enum class NotificationType {
```

Comment:
Total nit, but it might make the enum values easier to read here if we reversed the order of the words so that the scope appears first:
```kotlin
enum class NotificationType {
  RESOURCE_UNHEALTHY,
  ARTIFACT_PINNED,
  ARTIFACT_UNPINNED,
  ARTIFACT_MARKED_AS_BAD,
  APPLICATION_PAUSED,
  APPLICATION_RESUMED,
  LIFECYCLE_EVENT
}
```